inference-engine/src/multi_device/multi_device_exec_network.cpp

```diff
@@ -260,6 +263,13 @@ void MultiDeviceExecutableNetwork::TryToLoadNetWork(AutoLoadContext& context,
         return;
     }
 
+    // need to reload network, unregister it's priority
+    // there maybe potential issue.
+    // for example they are dGPU, VPUX, iGPU, customer want to LoadNetwork with
```

Comment:
@wangleis, please help to check here

Reply:
Could we have pipeline to load network one by one?